### PR TITLE
fix: express server should save rawBody

### LINKED ISSUE #2183: infrastructure: add rawBody on Cloud Run too
With #2176 gcf-utils started to use rawBody for signature verification on Cloud Function.
We should support the same thing on Cloud Run too.

### PR DESCRIPTION
**Pull request by chingor13**
Fixes #2183

### CODE REVIEW

File: packages/gcf-utils/test/server.ts [hunk 1]
```diff
@@ -168,3 +175,50 @@ describe('GCFBootstrapper', () => {
...
+    const response = await gaxios.request({
+      url: `http://localhost:${TEST_SERVER_PORT}/`,
+      headers: {
+        'x-github-delivery': '123',
+        'x-cloudtasks-taskname': 'test-bot',
+        'x-github-event': 'issues',
+        'content-type': 'application/json',
+      },
+      method: 'POST',
+      body: JSON.stringify(payload),
```
Comment (tmatsuo) [inline packages/gcf-utils/test/server.ts:220]: Can you add indent here and verify the buffer retains the indent?

**Comment by tmatsuo** [comment]
Can we also use this on Cloud Function safely too? FWIW, Cloud Function already automatically inject the rawBody.

**Comment by chingor13** [comment]
> Can we also use this on Cloud Function safely too? FWIW, Cloud Function already automatically inject the rawBody.

Cloud Functions runs its own wrapper server (as does functions-framework) which does this. We only need to do this for our wrapper `http.Server` implementation (for Cloud Run right now).

File: packages/gcf-utils/test/server.ts (hunk 1 above)
Comment (chingor13) [reply packages/gcf-utils/test/server.ts]: Modified the test server to return the rawBody in the response and we assert that the body indentation has not changed.

**Review by tmatsuo — APPROVED** [review]

**Pull request merged**